one-app-statics/README.md

```diff
@@ -0,0 +1,65 @@
+<h1 align="center">
+  <img src='https://github.com/americanexpress/one-app/raw/master/one-app.png' alt="One App - One Amex" width='50%'/>
+</h1>
+
+# One App Client Assets and CDN
+
+## 🚨 Important 🚨  
+
+This Package is not meant to be installed using npm, it only serves as the origin for the CDN for the static client assets for `one-app` once published to npm,
```

Comment:
is there any reason why we would care?

Reply:
@jsmartfo I think it is a nice touch in the odd chance someone finds the npm package and tries to install/require it 👍

Reply:
right, but say someone wanted to stand up the app using plain old statics, server-less, would we mind?

Reply:
I am not against it being in there as a warning, but we should say its not a supported method? after all I don't think we care how people use it.

Reply:
They can go to JSDelivr and download the statics from there, the warning is there so we don't get people trying to use it from npm instead of JSDelivr